pkg/apis/core/validation/validation.go

```diff
@@ -1601,6 +1602,9 @@ func ValidatePersistentVolumeClaimSpec(spec *core.PersistentVolumeClaimSpec, fld
 		allErrs = append(allErrs, field.Required(fldPath.Child("resources").Key(string(core.ResourceStorage)), ""))
 	} else {
 		allErrs = append(allErrs, ValidateResourceQuantityValue(string(core.ResourceStorage), storageValue, fldPath.Child("resources").Key(string(core.ResourceStorage)))...)
+		if storageValue.Value() == int64(0) {
```

Comment:
will this break any reasonable users?  It's technically a breaking change.

Reply:
I think it does not make sense to request 0 storage in PVC

Reply:
I know you think it doesn't make sense :).  The question was whether this will impact users.  If a user specified 0 today, what happens?  Does it do something reasonable or does it fail at some later point?

Reply:
We do pvc request value `not zero check` in external storage repo,  https://github.com/kubernetes-incubator/external-storage/blob/master/ceph/rbd/pkg/provision/rbd_util.go#L47
but we do not do that in core kubernetes repo. https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/rbd/rbd_util.go#L364

Reply:
In external repo, we do pvc request value `not zero check` in external storage repo,  https://github.com/kubernetes-incubator/external-storage/blob/master/ceph/rbd/pkg/provision/rbd_util.go#L47,  if a user requests 0, it will fail, but in kubernetes repo, we do not check this. will try to provision 0 capacity volume